webcompat/templates/issue.html

```diff
@@ -45,6 +45,9 @@
 {%- if config.PRODUCTION or config.STAGING -%}
 <script src="{{ url_for('static', filename='js/dist/issues.min.js')|bust_cache }}"></script>
 {% else %}
+<script src="{{ url_for('static', filename='js/vendor/markdown-it.js') }}"></script>
+<script src="{{ url_for('static', filename='js/vendor/markdown-it-emoji-1.4.0.js') }}"></script>
+<script src="{{ url_for('static', filename='js/vendor/markdown-it-sanitizer-0.4.1.js') }}"></script>
```

Comment:
Do we still need them?

Reply:
Unfortunately yes.
When a user types a comment we need to show it formatted correctly right after he sent the comment, so we need to format the comment ourselves.

Hmm... Or maybe, now that I'm thinking about it, we could make a request to Github right after the user post their comment, maybe it'd be too slow though. What do you think?

Reply:
ah understood. ok let's keep it for now.